tcms/testruns/templates/testruns/mutable.html

```diff
@@ -96,6 +96,33 @@
             </div>
         </div>
 
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_planned_stat">{% trans "Planned start" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_planned_stat" value="{{ form.planned_start }}">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar"></span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_planned_stat" %}
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_planned_stop">{% trans "Planned stop" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_planned_stop" value="{{ form.planned_stop }}">
```

Comment:
Mind the syntax in the value attribute.  As-is produces invalid HTML, see how the actual page looks like in the browser.

Also make sure that if there are errors they are displayed.

Reply:
Yes, I was not sure why the HTML was off yesterday, but today I figured that I need to use the object variable instead. Although the HTML looks okay now, seems like the value is not set since when I reopen the edit page it does not display the updated value.  Do you think I need to update something else?

Reply:
>  Do you think I need to update something else?

Yes, go back to using form fields and pay attention to the syntax of what you write and how other fields are written.